github/resource_github_organization_ruleset.go

```diff
@@ -63,11 +62,12 @@ func resourceGithubOrganizationRuleset() *schema.Resource {
 						},
 						"bypass_mode": {
 							Type:         schema.TypeString,
-							Optional:     true,
+							Required:     true,
```

Comment:
This is required in the GitHub API and so previous usage would just end in an error when the request is made, which is why it's a patch and not a breaking change.